content/02.main-text.md

```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
 These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
 GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
 
-`TODO: cgreene paragraph on diversity, open science culture, possibly author disagreements and conduct`
+In our open review paper, we established [contributor guidelines](https://github.com/greenelab/deep-review/blob/v0.9-preprint/CONTRIBUTING.md) that discussed norms in the areas of of text contribution, peer review, and authorship, which we identified in advance as potential areas of disagreement.
+Our contributor guidelines required verifiable participation: either directly attributable changes to the text or participation in the discussion on GitHub.
+We maintained our guidelines, even when a case arose where two authors had written text together but only one had directly attributable changes and participation.
+These guidelines did not discuss broader community norms that may have improved inclusiveness.
+It is also important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
+Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1080/08874417.2009.11645338].
```

Comment:
> Ok - I addressed everything except the DOI with issues. I'm not quite sure what to do there. I guess moving to the citation of the earlier paper seems like the cleanest way to go right now.

What about switching to the previous version and commenting in the source:

```md
Recent work has identified clear social norms and processes as helpful to maintaining a collaborative culture [@doi:10.1109/hicss.2007.277].
<!-- cited a previous version of @doi:10.1080/08874417.2009.11645338 (see https://git.io/vdSSR) -->
```

Reply:
I agree with this 👍  - done!

Reply:
From @cgreene:

> manubot appears to try to build the @doi: etc even if it's in a comment
> https://travis-ci.org/greenelab/meta-review/builds/289122620

Yes I'll add HTML comments the the snippets that should be ignored in https://github.com/greenelab/manubot/issues/13.

Removing `@` or `@doi:` as was done in https://github.com/greenelab/meta-review/pull/12/commits/370e6d79be7cd387eac6c7be6fca8b94ecca4b04 is a good workaround.